cassandra/README.md

```diff
@@ -7,343 +7,149 @@
-_cloud native_, we mean an application which understands that it is running
-within a cluster manager, and uses this cluster management infrastructure to
-help implement the application.  In particular, in this instance, a custom
-Cassandra `SeedProvider` is used to enable Cassandra to dynamically discover
-new Cassandra nodes as they join the cluster.
-
-This example also uses some of the core components of Kubernetes:
-
-- [_Pods_](/docs/user-guide/pods)
-- [ _Services_](/docs/user-guide/services)
-- [_Replication Controllers_](/docs/user-guide/replication-controller)
-- [_Stateful Sets_](/docs/concepts/workloads/controllers/statefulset/)
-- [_Daemon Sets_](/docs/admin/daemons)
-
-## Prerequisites
-
-This example assumes that you have a Kubernetes version >=1.2 cluster installed and running,
-and that you have installed the [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-command line tool somewhere in your path.  Please see the
-[getting started guides](https://kubernetes.io/docs/getting-started-guides/)
-for installation instructions for your platform.
-
-This example also has a few code and configuration files needed.  To avoid
-typing these out, you can `git clone` the Kubernetes repository to your local
-computer.
-
-## Cassandra Docker
+---
+title: Using StatefulSets with Cassandra
+---
 
+{% capture overview %}
+This tutorial shows you how to develop a native cloud [Cassandra](http://cassandra.apache.org/) deployment on Kubernetes. In this instance, a custom Cassandra `SeedProvider` enables Cassandra to discover new Cassandra nodes as they join the cluster.
+
+**Cassandra Docker**
 The pods use the [```gcr.io/google-samples/cassandra:v12```](https://github.com/kubernetes/examples/blob/master/cassandra/image/Dockerfile)
 image from Google's [container registry](https://cloud.google.com/container-registry/docs/).
-The docker is based on `debian:jessie` and includes OpenJDK 8. This image
-includes a standard Cassandra installation from the Apache Debian repo.  Through the use of environment variables you are able to change values that are inserted into the `cassandra.yaml`.
+The docker is based on `debian:jessie` and includes OpenJDK 8. This image includes a standard Cassandra installation from the Apache Debian repo.  By using environment variables you can change values that are inserted into `cassandra.yaml`.
 
 | ENV VAR       | DEFAULT VALUE  |
 | ------------- |:-------------: |
 | CASSANDRA_CLUSTER_NAME | 'Test Cluster'  |
 | CASSANDRA_NUM_TOKENS  | 32               |
 | CASSANDRA_RPC_ADDRESS | 0.0.0.0          |
 
-## Quickstart
-
-If you want to jump straight to the commands we will run,
-here are the steps:
-
-```sh
-#
-# StatefulSet
-#
+{% endcapture %}
 
-# clone the example repository
-git clone https://github.com/kubernetes/examples
-cd examples
+{% capture objectives %}
+* Create and Validate a Cassandra headless `Service`.
+* Use a StatefulSet to create a Cassandra ring.
+* Validate the StatefulSet.
```

Comment:
if you're escaping `Service`, I recommend doing so with `StatefulSet` and`Pod` too.